content/blog/command-provider-ga/index.md

```diff
@@ -0,0 +1,750 @@
+                            Map.entry("FN", arn),
+                            Map.entry("AWS_REGION", awsRegion),
+                            Map.entry("AWS_PAGER", "")))
+                    .build(),
+                    CustomResourceOptions.builder()
+                            .dependsOn(lambdaFunction)
+                            .build());
+
+            return invokeCommand.stdout();
+        });
+
+        ctx.export("output", lambdaInvokeOut);
+    }
+}
+```
+
+{{% /choosable %}}
+
+{{% choosable language yaml %}}
+
+```yaml
+
+resources:
+  lambdaRole:
+    type: aws:iam:Role
+    properties:
+      assumeRolePolicy:
+        fn::toJSON:
+          Version: "2012-10-17"
+          Statement:
+            - Action: sts:AssumeRole
+              Effect: Allow
+              Principal:
+                Service: lambda.amazonaws.com
+
+  lambdaFunction:
+    type: aws:lambda:Function
+    properties:
+      name: f
+      publish: true
+      role: ${lambdaRole.arn}
+      handler: index.handler
+      runtime: "nodejs20.x"
+      code:
+        fn::fileArchive: ./handler
+
+  invokeCommand:
+    type: command:local:Command
+    properties:
+      create: 'aws lambda invoke --function-name "$FN" --payload ''{"stackName": "${pulumi.stack}"}'' --cli-binary-format raw-in-base64-out out.txt >/dev/null && cat out.txt | tr -d ''"''  && rm out.txt'
+      environment:
+        FN: ${lambdaFunction.arn}
+        AWS_REGION: ${aws:region}
+        AWS_PAGER: ""
+    options:
+      dependsOn:
+        - ${lambdaFunction}
+
+outputs:
+  output: ${invokeCommand.stdout}
+```
+
+{{% /choosable %}}
+
+## What's new
+
+The 1.0 release of the Command provider marks a stable API for the 1.x series. Rather than simply putting a “v1” label on the existing provider, we also used this opportunity to enhance it with some previously requested capabilities to provide a more complete set of capabilities.
+
+- The API documentation in the Pulumi registry has [examples in all Pulumi languages](https://github.com/pulumi/pulumi-command/issues/196) and is expanded.
+- Capturing stdout and stderr of commands can now [be switched off](https://github.com/pulumi/pulumi-command/pull/451), which is useful when they might contain secrets or are very noisy.
+- Environment handling for remote commands was [fixed and is better documented](https://github.com/pulumi/pulumi-command/pull/395).
+- The `CopyFile` resource is [superseded](https://github.com/pulumi/pulumi-command/pull/423) by the new `CopyToRemote` resource. It can copy whole directories in addition to individual files. The source of the copy is now a [Pulumi asset or archive](https://www.pulumi.com/docs/concepts/assets-archives/) which provides full interoperability with the Pulumi ecosystem. The use of assets and archives also makes Pulumi run copy operations only if the source has changed. For an easy transition, the previous `CopyFile` resource will remain available for a while with a deprecation notice.
```

Comment:
rather than saying "for a while" why don't we commit to keeping it around until the next major version of command? (We know this will hold until then since we don't like to make a breaking change like that in a minor and is sounds a little more reassuring)